packages/api/src/authentication/jwt-token.ts

```diff
@@ -0,0 +1,26 @@
+import * as jose from "jose";
+import { getJwtPayload } from "../bridge/token-data.js";
+import { JwtPayload } from "./types/jwt-payload.js";
+
+async function generateJwtToken(payload: JwtPayload): Promise<string | null> {
+  const decodedSessionKey = jose.base64url.decode(payload.sessionKey);
+
+  return new jose.SignJWT({
+    ...payload.userInfo,
+    sessionId: payload.sessionId,
+  })
+    .setProtectedHeader({ alg: "HS256" })
+    .setIssuedAt()
+    .setExpirationTime("60s")
+    .sign(decodedSessionKey);
+}
+
+export async function getToken(): Promise<string | null> {
```

Comment:
If it's server side you can use `JWTAuthentication` from `@orochi-network/framework`

Reply:
It's on client side